src/main/java/esubine/community/auth/model/TokenRepository.java

```diff
@@ -7,7 +7,15 @@
 public interface TokenRepository extends JpaRepository<TokenEntity, Long> {
     TokenEntity findByToken(String token);
 
+//    @Modifying
+//    @Query(value = "delete from token where user_id=:userId", nativeQuery = true)
+//    void deleteAllByUserId(Long userId);
+
     @Modifying
-    @Query(value = "delete from token where user_id=:userId", nativeQuery = true)
-    void deleteAllByUserId(Long userId);
+    @Query("update TokenEntity t set t.isDelete=true " +
+            "WHERE t.userId=:userId ")
+    void setAllByUserId(Long userId);
```

Comment:
지금 네임이은 userId를 설정하겠다는 이야기인데
userId를 이용해서(by) userId를 설정한다? 말이 이상하죠
```suggestion
    void deleteAllByUserId(Long userId);
```